content/docs/dvclive/api-reference/live/log_image.md

```diff
@@ -0,0 +1,77 @@
+# Live.log_image()
+
+Saves the given image `val` to the output file `name`.
+
+```py
+def log_image(name: str, val):
+```
+
+#### Usage:
+
+```py
+from dvclive import Live
+
+live = Live()
+
+import numpy as np
+img_numpy = np.ones((500, 500), np.uint8) * 255
+live.log_image("numpy.png", img_numpy)
+
+from PIL import Image
+img_pil = Image.new("RGB", (500, 500), (250, 250, 250))
+live.log_image("pil.png", img_pil)
+```
+
+## Description
+
+Supported values for `val` are:
+
+- A valid numpy array (convertible to image via
+  [PIL.Image.fromarray](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.fromarray))
+- A `PIL.Image` instance.
+
+The images will be saved in `{path}/images/{name}`:
```

Comment:
First time `path` is referenced. Unclear where this comes from, should link to the DVClive config or something, I think.

Reply:
p.s. feel free to contribute to #3304 to address any of these comments if you agree @daavoo .